scripts/train.ipynb

```diff
@@ -75,111 +174,258 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "UkNPbx2MYmGe"
+   },
    "outputs": [],
    "source": [
-    "from wandb_logger import log_results, save_params"
+    "# > VS CODE SPECIFIC <\n",
+    "# Uncoment if run in VS Code embeded notebook\n",
+    "# import warnings\n",
+    "# Needed to show warnings in VS Code integrade jupyter\n",
+    "# warnings.simplefilter(action=\"default\")"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "z6ZseXVHYmGe"
+   },
    "outputs": [],
    "source": [
-    "# DEBUG: Tools to export networkx graph\n",
-    "from networkx import to_dict_of_lists, to_dict_of_dicts"
+    "# Enable reproducibility\n",
+    "# https://pytorch.org/docs/stable/notes/randomness.html\n",
+    "np.random.seed(42)\n",
+    "torch.manual_seed(42)\n",
+    "# Unable to use below command with True as non-deterministics algos are used by CUDA\n",
+    "torch.set_deterministic(False)"
```

Comment:
This is only available from torch 1.7, so you need to update the requirements (I only had the 1.6 version installed).

Also this is a beta feature for now.

Reply:
And if it's not useful in the current state I suggest just removing it

Reply:
ok to remove.